PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -493,6 +495,7 @@ void AliEmcalJetTask::FillJetConstituents(AliEmcalJet *jet, std::vector<fastjet:
         AliError(Form("Could not find track %d",tid));
         continue;
       }
+      jet->AddParticleConstituent(t);
```

Comment:
The embedding status should be set here as part of the function call. Should probably be something like:

~~~c++
jet->AddParticleConstituent(t, partCont->GetIsEmbedding());
~~~

Of course, that will require the modification of some constructors